ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -124,6 +123,11 @@ const triggerSet: TriggerSet<Data> = {
       type: 'StartsUsing',
       // 62D4 = Holiest of Holy
       // 63C8 = Ascalon's Mercy Concealed
+      // 6708 = Final Chorus
+      // 62E2 = Spear of the Fury
+      // 6B87 = the Dragon's Eye
+      // 6667 = unknown_6667
+      // 7438 = Alternative End
       netRegex: NetRegexes.startsUsing({ id: ['62D4', '63C8'], capture: true }),
```

Comment:
Can you make the regex include these ids as well?

Reply:
Thanks for reviewing it, forgot about the regex portion.